### DISCUSSION TITLE
graphscope运行环境问题

### DISCUSSION THREAD

**Discussion by Maxismcaptain** [alibaba/GraphScope#3631] [Q&A] [answered]
最近在华为鲲鹏服务器和麒麟服务器上部署graphscope，总会出现各种各样的错误，无法正常部署，但是在Ubuntu虚拟机上即可正常运行。请问是否是因为graphscope不支持aarch64架构，只能在x86架构上运行，graphscope具体的环境要求是怎样的呢？
![1ad44e6b61f883a0d7a661ee09ee5df](https://github.com/alibaba/GraphScope/assets/70937431/04db2f33-446c-4692-ae07-75716cd3e1eb)

**Comment by siyuan0322** [comment]
我们发布了 amd64 和 Darwin python wheel 包，以及 amd64 和 arm64 的镜像。图中的错误属于没有 arm 包的问题。请问你是打算在服务器上使用呢，还是想试用一下

**Reply by Maxismcaptain** [reply]
我们希望对graphscope做增量开发，支持一些新的功能，运行在国产化的平台上，请问上述arm包如何获得呢，aarch64架构是否也无法使用呢

**Comment by lidongze0629** [comment, marked as answer]
@Maxismcaptain 你好，这是 0.24 版本 client wheel 支持的平台列表，里面是包含 aarch64 包的，请问咱们这边的 compatible_wheels 格式是什么呢？

**Reply by Maxismcaptain** [reply]
您好老师，我们也不太了解这个compatible_wheels，我们是想对graphscope的源码做一些改动，类似spark一样编译出jar包然后使用，您发的aarch64包应该是安装包吧，请问有关源码编译有相关的文档或操作指南吗

**Reply by siyuan0322** [reply]
可以从服务器本地编译安装  https://github.com/alibaba/GraphScope/blob/main/docs/development/dev_guide.md

**Reply by Maxismcaptain** [reply]
好的，谢谢您！

**Comment by lidongze0629** [comment]
[compatible_wheels](https://pypi.org/project/graphscope-client/0.24.0/#files)
